ieee-quantum-week/bqskit.ipynb

```diff
@@ -0,0 +1,580 @@
+    "compiled_mitigated_result = zne.execute_with_zne(compiled_circuit, execute)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65242ae6",
+   "metadata": {},
+   "source": [
+    "## Comparison\n",
+    "\n",
+    "These data are then summarized in the following output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 124,
+   "id": "97266f1a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "            uncompiled      compiled  \n",
+      "ideal        0.19            0.19      \n",
+      "noisy        0.16            0.16      \n",
+      "mitigated    0.18            0.19      \n",
+      "error        0.01            0.00      \n",
+      "depth        10              49        \n"
+     ]
+    }
+   ],
+   "source": [
+    "header = \"{:<11} {:<15} {:<10}\"\n",
+    "entry = \"{:<11}  {:<15.2f} {:<10.2f}\"\n",
+    "int_entry = \"{:<11}  {:<15} {:<10}\"\n",
+    "print(header.format(\"\", \"uncompiled\", \"compiled\"))\n",
+    "print(entry.format(\"ideal\", uncompiled_ideal_value, compiled_ideal_value))\n",
+    "print(entry.format(\"noisy\", uncompiled_noisy_value, compiled_noisy_value))\n",
+    "print(entry.format(\"mitigated\", uncompiled_mitigated_result, compiled_mitigated_result))\n",
+    "print(entry.format(\n",
+    "    \"error\",\n",
+    "    abs(uncompiled_ideal_value - uncompiled_mitigated_result),\n",
+    "    abs(compiled_ideal_value - compiled_mitigated_result),\n",
+    "))\n",
+    "print(int_entry.format(\n",
+    "    \"depth\",\n",
+    "    len(random_circuit),\n",
+    "    len(compiled_circuit),\n",
+    "))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "619c30fb",
+   "metadata": {},
+   "source": [
+    "Hence for this particular quantum volume circuit we see that using both compilation _and_ error mitigation combine for the most accurate result.\n",
+    "Note that that depth of the compiled circuit is much larger than that of the initial random circuit.\n",
+    "This is caused by the fact that quantum volume circuits consist of \"matrix gates\" which encode much more complex operations than the gateste that `bqskit` compiles into by default.\n",
+    "Hence it is expected to see an increase in depth for most quantum volume circuits.\n",
+    "For different types of random circuits (e.g. `cirq`'s [`random_circuit`](https://quantumai.google/reference/python/cirq/testing/random_circuit) function seen in the next section), this is not the case.\n",
+    "\n",
+    "We can now wrap this code into a loop to get a better understanding of how errors change for the following four combinations of compilation and mitigation.\n",
+    "\n",
+    "| compiled | mitigated |\n",
+    "| -------- | --------- |\n",
+    "| ❌       | ❌         |\n",
+    "| ❌       | ✅         |\n",
+    "| ✅       | ❌         |\n",
+    "| ✅       | ✅         |"
```

Comment:
What is the meaning of this table?

Reply:
Just trying to make it clear that there are four possible ways to "combine" compilation and mitigation and that we are going to compare them. If it doesn't help, happy to remove.

Reply:
Ah, now I see. Fine to keep it